src/web_app_skeleton/src/actions/api_action.cr.ecr

```diff
@@ -12,4 +13,5 @@ abstract class ApiAction < Lucky::Action
   # Add 'include Api::Auth::SkipRequireAuthToken' to your actions to allow all requests.
   include Api::Auth::RequireAuthToken
   <%- end -%>
+  include Lucky::EnforceUnderscoredRoute
```

Comment:
@paulcsmith is it intentional that this is included twice in `ApiAction`? It doesn't seem to be inside of an `if` block or anything that would require it.

Reply:
oh, good call. I missed that... Hmm 🤔  Yeah it seems this last one isn't necessary.

Reply:
Yeah it was an accident. Nice catch! Will remove